deployments/charts/kuma-cp/create-namespace.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+NAMESPACE=${1:-kuma-system}
+
+kubectl create namespace "$NAMESPACE" || true
+
+kubectl patch namespace/"$NAMESPACE" \
+  --type merge \
+  --patch '{ "metadata": { "labels": { "kuma.io/system-namespace": "true" } } }'
```

Comment:
Why do you need this one?
 Can you just apply this one -> https://github.com/kumahq/kuma/blob/master/app/kumactl/data/install/k8s/control-plane/namespace.yaml

Reply:
Is there a way with `kumactl` to install only the namespace? Helm 3 requires a namespace to exist before deployment:

```
$ helm3 install kuma-cp . --namespace kuma-system
Error: create: failed to create: namespaces "kuma-system" not found
```


The other way would be to make users specify the namespace in `values.yaml` and ignore the Helm `.Release.Namespace` entirely, though this is not ideal in terms of fitting with the Helm conventions and would likely lead to confusion.

Reply:
no ! right now, there is no way to install namespace only through `kumactl`. Can we
keep the `kuma-system` as a default value in the namespace ?

Reply:
Hey @tharun208, can you elaborate on the ask? I'm not sure I understand. The issue w/ Helm 3 is that you must create the namespace, whether it is `kuma-system` or another`, before the actual deployment.

Reply:
Following up from slack, we've tried to use the Helm 3.2.0 option `--create-namespace` but results in the same error.

Reply:
Another hacky option here would be to run a Job w/ [a pre-install ](https://helm.sh/docs/topics/charts_hooks/#the-available-hooks) that updates the namespace with the label using `kubectl` directly.

Reply:
@nickolaev, what do you think about this approach of adding the label with a pre-install job?

Reply:
Why would this be "hacky"? It looks legitimate to me? I am surprised they did not make that the default behaviour.

Reply:
Just not something I've seen before, but happy to give it a shot. A bit like the "jump host" solution you were talking about

Reply:
It worked! Added in the latest commit